<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.994, stop:0 rgba(207, 236, 208, 255), stop:1 rgba(255, 197, 202, 255))</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QPushButton" name="Reshit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 247, 216);
color: rgb(0,0,0);
border-radius: 20px</string>
    </property>
    <property name="text">
     <string>go!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="another">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>450</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 247, 216);
color: rgb(0,0,0);
border-radius: 20px</string>
    </property>
    <property name="text">
     <string>another</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="arg1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,30);
color: rgb(0,0,0);
border-radius: 10px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="arg2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,30);
color: rgb(0,0,0);
border-radius: 10px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="arg3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,30);
color: rgb(0,0,0);
border-radius: 10px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Zagolovok">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: none;

color: rgb(0,0,0);

</string>
    </property>
    <property name="text">
     <string>Assignment</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Otvet">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,30);
color: rgb(0,0,0);
border-radius: 10px</string>
    </property>
    <property name="text">
     <string>Unswer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
